src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,12 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
+    public static User findByEmail(String email) {
+        if (email.equals(users[0].getEmail())) {
+            return users[0];
+        } else if (email.equals(users[1].getEmail())) {
+            return users[1];
+        }
```

Comment:
what if your users array contains more than 2 users in it? I'd recommend using loop here